pages/ksphere/kommander/latest/introducing-kommander/index.md

```diff
@@ -7,15 +7,33 @@ excerpt: Deploy and manage Kubernetes on an enterprise scale
 category: K-Sphere
 menuWeight: 2
 ---
+Kommander delivers visibility and management of any Kubernetes cluster, whether on-prem or cloud, regardless of distribution being used. Organizations can gain better control over existing deployments without service interruptions and create standardization around how new clusters are configured and used.
 
-Kommander lets you manage multiple Kubernetes clusters across your entire deployment quickly and seamlessly. Kommander provides a centralized console for deploying, monitoring, and managing production-ready Kubernetes on an enterprise scale.
+Kommander was developed to address the broad issues caused by Kubernetes “cluster sprawl,” D2iQ’s Kommander delivers federated management, governance, and visibility across an organization’s expansive use of Kubernetes clusters. As various parts of the organization require new Kubernetes clusters and associated data services, Kommander simplifies the provisioning of new services and creates greater policy-driven consistency across Kubernetes clusters within the environment. This capability allows for greater standardization of Kubernetes ensuring easier support paths for application services. This level of broad control helps organizations meet risk and compliance demands as they govern how and where new application services are used, and who is able to engage in policy and operational needs of those services.
```

Comment:
```suggestion
Kommander was developed to address the broad issues caused by Kubernetes “cluster sprawl”. D2iQ’s Kommander delivers federated management, governance, and visibility across an organization’s expansive use of Kubernetes clusters. As various parts of the organization require new Kubernetes clusters and associated data services, Kommander simplifies the provisioning of new services and creates greater policy-driven consistency across Kubernetes clusters within the environment. This capability allows for greater standardization of Kubernetes ensuring easier support paths for application services. This level of broad control helps organizations meet risk and compliance demands as they govern how and where new application services are used, and who is able to engage in policy and operational needs of those services.
```

Reply:
Oh, weird, this is showing the entire passage, rather than the diff. I only changed the comma to a period in the sentence
```
Kommander was developed to address the broad issues caused by Kubernetes “cluster sprawl”.
```